docs/examples/cart-join.md

```diff
@@ -118,42 +117,43 @@ def format_output(user_id__joined_state):
 flow.map(format_output)
 ```
 
-And finally, capture this output and send it to our output builder.
-
-```python
-flow.capture()
-```
 
 On that note, we'll serialize the final output as JSON and print it.
 
 ```python
 def output_builder(worker_index, worker_count):
-    def output_handler(epoch_item):
-        epoch, item = epoch_item
+    def output_handler(item):
         line = json.dumps(item)
-        print(epoch, line)
+        print(line)
```

Comment:
Maybe two captures in the future 🤔 Seems odd that we're not employing the `StdOut`

Reply:
I think we have to capture the stdout (like in the tests) for it to run successfully. that's arguably more awkward so I'm fine leaving it. We can use `StdOutputConfig` in the py examples still